tests/unittests/test_error_messages.py

```diff
@@ -28,7 +28,7 @@ def test_error_1(self, mocked_post_request):
             "client_secret": "client_secret_123",
             "tenant_subdomain": "",
             "start_date": "2019-01-01T00:00:00Z",
-            "request_timeout": "900",
+            "request_timeout": "300",
```

Comment:
Why the timeout was changed from 900 to 300? There must be a reason it was set to 900.

Reply:
@KrisPersonal To maintain consistency among all the Tap, we have asked before implementing.
Please find the reference link: https://talend.slack.com/archives/C01TGC4QAUE/p1635252663017600